platforms.go

```diff
@@ -121,7 +121,8 @@ import (
 )
 
 var (
-	specifierRe = regexp.MustCompile(`^[A-Za-z0-9_-]+$`)
+	specifierRe        = regexp.MustCompile(`^[()A-Za-z0-9_.-]+$`)
```

Comment:
> Not sure how strict we want the `<OSVersion>` to be; if we only expect the versions as used by windows (one or more dot-separated digits), then it could be;
>
Not sure we want to enforce this keeping only windows in mind as the format for different OS-es could be different. I think we should take in whatever is honored and have the platform matcher or smth else which actually reads or uses this field to error out if this is not valid.